src/Core/Content/Media/Extension/ResolveRemoteThumbnailUrlExtension.php

```diff
@@ -0,0 +1,35 @@
+<?php
+declare(strict_types=1);
+
+/*
+ * Created by netlogix GmbH & Co. KG
+ *
+ * @copyright netlogix GmbH & Co. KG
+ */
+
+namespace Shopware\Core\Content\Media\Extension;
+
+use Shopware\Core\Framework\Extensions\Extension;
+use Shopware\Core\Framework\Log\Package;
+
+/**
+ * @extends Extension<string>
```

Comment:
can you add codecoverageignore here as phpdoc

Reply:
Done